src/tf/TFClient.js

```diff
@@ -45,15 +64,16 @@ function TFClient(options) {
  *
  * @param tf - the TF message from the server
  */
-TFClient.prototype.processFeedback = function(tf) {
+TFClient.prototype.processTFArray = function(tf) {
   var that = this;
   tf.transforms.forEach(function(transform) {
     var frameID = transform.child_frame_id;
-    if (frameID[0] === '/') {
+    if (frameID[0] === '/')
+    {
       frameID = frameID.substring(1);
```

Comment:
Since it's only used in this file, and only three times, I don't think it's necessary to import a module just for trimLeft. Also, I believe using built-in functions keeps the code a tiny bit more accessible.

What other string functions would be useful for us? If there's other areas where we can simplify code significantly, of course we should consider it.